### PR TITLE
issue template updates

### PR DESCRIPTION
**Pull request by herbig**
## Description
Fixed a typo and made some modifications, now that I can actually see what this looks like..

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bugs.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: Bug
 description: Report a bug
-title: "[Bug]: "
+title: "Bug: "
```
Comment (herbig) [inline .github/ISSUE_TEMPLATE/bugs.yml:3]: Didn't like these brackets..

File: .github/ISSUE_TEMPLATE/bugs.yml [hunk 2]
```diff
@@ -20,7 +20,9 @@ body:
     attributes:
       label: Steps to reproduce
       description: How can we replicate the issue?
-      placeholder: 1. Navigate to the homepage \n 2. Click Create DAO \n 3. My keyboard bursts into flames
+      placeholder: 1. Navigate to the homepage
+      2. Click Create DAO
+      3. My keyboard bursts into flames
```
Comment (herbig) [inline .github/ISSUE_TEMPLATE/bugs.yml:25]: Newline didn't work, so maybe this will... We'll see

File: .github/ISSUE_TEMPLATE/bugs.yml [hunk 3]
```diff
@@ -67,16 +69,16 @@ body:
         - iOS
         - Android
     validations:
-      required: true
+      required: false
```
Comment (herbig) [inline .github/ISSUE_TEMPLATE/bugs.yml:72]: app version and browser are required, but I made OS and wallet not required, just to make it slightly easier to report and those aren't as important

Comment (SudoPumpkin) [reply .github/ISSUE_TEMPLATE/bugs.yml:72]: Yeah, I like having those fields there for the user just to trigger a 'hey, these data points would be great for repro.' Not to be required though, good call.

File: .github/ISSUE_TEMPLATE/bugs.yml (hunk 2 above)
Comment (Da-Colon) [reply .github/ISSUE_TEMPLATE/bugs.yml:25]: https://yaml-multiline.info/

Comment (herbig) [reply .github/ISSUE_TEMPLATE/bugs.yml:28]: Ah thank you sir, updated

**Review by Da-Colon — APPROVED** [review]

**Pull request merged**